dashboard/app/reporting_lists.go

```diff
@@ -349,6 +346,13 @@ func queryMatchingBugs(c context.Context, ns, name string) ([]*Bug, []*Bug, erro
 			fixed = append(fixed, bug)
 			continue
 		}
+		currReporting, _, _, _, err := currentReporting(c, bug)
+		if err != nil {
+			continue
+		}
+		if currReporting.AccessLevel != AccessPublic {
```

Comment:
We need to select bugs that match visibility of the target stage. These are not necessary public, there may be no public bugs in a namespace at all.

Reply:
Fixed. PTAL.